enabler/src/de/schildbach/pte/NetworkProvider.java

```diff
@@ -160,8 +188,8 @@ QueryTripsResult queryTrips(Location from, @Nullable Location via, Location to,
      * 
      * @param context
      *            context to query more trips from
-     * @param next
```

Comment:
This fix seems unrelated to the PR. Please don't mix.

Reply:
Ok, I'll change that.

Reply:
Yes, please do. You can fix it in a separate PR if you want, then I'll merge the JavaDoc fix right away.